query-translator/tests/translate.rs

```diff
@@ -466,3 +466,39 @@ fn test_order_by() {
                      ORDER BY `?y_value_type_tag` ASC, `?y` ASC, `?x` ASC");
     assert_eq!(args, vec![]);
 }
+
+#[test]
+fn test_complex_nested_or_join_type_projection() {
+    let mut schema = Schema::default();
+    associate_ident(&mut schema, NamespacedKeyword::new("page", "title"), 98);
+    add_attribute(&mut schema, 98, Attribute {
+        value_type: ValueType::String,
+        ..Default::default()
+    });
+    associate_ident(&mut schema, NamespacedKeyword::new("page", "title2"), 97);
```

Comment:
This extra attribute isn't actually used in the test.  Remove it or use it -- see the comment below.